keras_cv/bounding_box/ensure_tensor.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.utils import preprocessing
+
+
+def ensure_tensor(boxes, dtype=None):
```

Comment:
This same method name exists in keras_cv.utils with different functionality -- do we want that? Should this be renamed?

Reply:
I believe we do that for a ton of `bounding_box` functions.  From my perspective we should make bounding boxes behave as closely to tensors as possible, while still being dictionaries.